src/plugins/intel_cpu/src/infer_request.h

```diff
@@ -81,7 +82,7 @@ class SyncInferRequest : public ov::ISyncInferRequest {
         }
 
         void update() {
-            m_proxyMemMngr->setMemMngr(currentMemMngr());
+            m_proxyMemMngr->setMemMngrResize(currentMemMngr());
```

Comment:
Why cannot the previous setMemMngr work now?

Reply:
Because it calls resize unconditionally, but this applicable in not all the contexts. Especially in the case when we transfer the input memory. Sometimes the input memory size is less, than the previous one, and the old logic would trigger resize unconditionally with loosing the underlining memory values.